rosdep/python.yaml

```
@@ -4542,6 +4542,19 @@ python-slycot-pip:
 python-smbus:
   debian: [python-smbus]
   ubuntu: [python-smbus]
+python-smbus2-pip:
```

```suggestion
python3-smbus2-pip:
```

pip rules should start with python3
Accepting this will then also require fixing the alphabetization too.